EIPS/eip-2917.md

```diff
@@ -0,0 +1,165 @@
+---
+eip: 2917
+title: Staking Reward Calculation
+author: Tony Carson <tony.carsonn@gmail.com>, Mehmet Sabir Kiraz <m.kiraz@gmail.com>, Süleyman Kardaş <skardas@gmail.com>
+discussions-to: https://github.com/ethereum/EIPs/issues/2917
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-08-28
+---
+
+## Simple Summary
+ERC3000 is a new standardization for on-chain calculation of staking reward.
```

Comment:
`ERC3000` it will be confusing to readers if this isn't the actual EIP number.  People have tried to snag vanity EIP numbers in the past and we have disallowed it because it leads to an arms race that just clogs up the EIP process.  My recommendation is to come up with a snazzy title rather than trying to get a vanity number.

Same comment for the other references to `ERC3000`.